sdk/features/arm-features/CHANGELOG.md

```diff
@@ -0,0 +1,16 @@
+## 30.0.0-beta.1 (Unreleased)
+
+This is beta preview version.
+
+This version uses a next-generation code generator that introduces important breaking changes, but also important new features (like unified authentication and async programming).
+
+**General breaking changes**
+
+- Authentication system has been completely revamped:
+
+  - Package `@azure/ms-rest-nodeauth` or `@azure/ms-rest-browserauth` are no longer supported, use package `@azure/identity` instead: https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/identity/identity
+
+- Operations with prefix `begin` like `beginXXX` that used to return a `Promise<Models.XXX>` now returns a poller that implements the `PollerLike` interface, and if you want to get previous result, please use operation name with prefix `begin` and suffix `AndWait`, such as `beginXXXAndWait`.
```

Comment:
I do not see LROs in this package:
```suggestion
```

Reply:
The changelog is a template, I think we need to add this block based on whether there is lro in the sdk.